index.js

```diff
@@ -10,7 +12,9 @@ module.exports = (robot) => {
   });
 
   robot.on('issue_comment.created', async context => {
-    if (context.isBot === false) {
+    if (
+      whitelistedAccounts.includes(context.repo().owner.toLowerCase()) &&

```

Comment:
I think having strict case checking is better here, otherwise it is still susceptible to abuse.

Reply:
Actually `.includes` does strict checking in the background. It uses `SameValueZero` comparison.
For reference, please see this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
then why is there a call to `.toLowerCase()`?

Reply:
So that even if the env variable, `WHITELISTED_ACCOUNTS` is defined as `oppia` or `OPPIA`, the check always considers it as `oppia`. The process followed here is similar to that followed at Probot: https://github.com/probot/scheduler/blob/master/index.js#L36